raccoon_src/utils/request_handler.py

```diff
@@ -91,12 +90,8 @@ def send(self, method="GET", *args, **kwargs):
         proxies = self._get_request_proxies()
 
         try:
-            if method.lower() == "get":
-                return requests.get(proxies=proxies, headers=self.headers, cookies=self.cookies, *args, **kwargs)
-            elif method.lower() == "post":
-                return requests.post(proxies=proxies, headers=self.headers, cookies=self.cookies, *args, **kwargs)
-            elif method.lower() == "head":
-                return requests.head(proxies=proxies, headers=self.headers, cookies=self.cookies, *args, **kwargs)
+            if method.upper() in ['GET', 'POST', 'HEAD']:
+                return request(method, timeout=5, proxies=proxies, headers=self.headers, cookies=self.cookies, *args, **kwargs)
```

Comment:
setting timeout to 5 seconds, shouldn't refactor outside because it's a formula of number of threads/CPUs and CPU usage.. http request above 5 seconds in a recon tool should be thrown out unless you are taking a screenshot and checking if the page is fully loaded
you have couple of requests that use time=15/20, test if they will contain timeout=5 or **kwargs will not override and exception will be thrown.. will fix it